photon-ml/src/main/scala/com/linkedin/photon/ml/supervised/classification/LogisticRegressionAlgorithm.scala

```diff
@@ -18,27 +18,35 @@ import breeze.linalg.Vector
 import com.linkedin.photon.ml.data.{LabeledPoint, ObjectProvider}
 import com.linkedin.photon.ml.function.{LogisticLossFunction, TwiceDiffFunction}
 import com.linkedin.photon.ml.normalization.NormalizationContext
-import com.linkedin.photon.ml.optimization.{LBFGS, Optimizer, OptimizerConfig, OptimizerFactory, RegularizationContext}
+import com.linkedin.photon.ml.optimization._
 import com.linkedin.photon.ml.supervised.model.GeneralizedLinearAlgorithm
 
 /**
- * Train a classification model for Logistic Regression.
+ * Train a classification model for Logistic Regression using L1/L2/Elastic net-regularized linear regression.
+ *
  * @note Labels used in Logistic Regression should be {0, 1}
- * @author xazhang
- * @author dpeng
  */
 class LogisticRegressionAlgorithm
   extends GeneralizedLinearAlgorithm[LogisticRegressionModel, TwiceDiffFunction[LabeledPoint]] {
 
   /**
-   *  Objective function = loss function + l2weight * regularization
-   *  Only the L2 regularization part is implemented in the objective function. L1 part is implemented through the
-   *  optimizer. See [[LBFGS]].
+   * Create the objective function of the generalized linear algorithm.
+   *
+   * objective function = loss function + l2weight * regularization
+   *
+   * Only the L2 regularization part is implemented in the objective function. L1 part is implemented through the
+   * optimizer. See [[LBFGS]].
+   *
+   * @param normalizationContext The normalization context for the training
+   * @param regularizationContext The type of regularization to construct the objective function
+   * @param regularizationWeight The weight of the regularization term in the objective function
    */
   override protected def createObjectiveFunction(
       normalizationContext: ObjectProvider[NormalizationContext],
       regularizationContext: RegularizationContext,
-      regularizationWeight: Double): TwiceDiffFunction[LabeledPoint] = {
+      regularizationWeight: Double)
+    : TwiceDiffFunction[LabeledPoint] = {
```

Comment:
Is there style guidance for this somewhere? I've usually seen it written the previous way, unless it doesn't fit on the line.